docs/api/resources/storage.rst

```diff
@@ -893,6 +893,33 @@ Re-Key
    :statuscode 202: no error
 
 
+Key Passphrase
+++++++++++++
+
+.. http:post:: /api/v1.0/storage/volume/(int:id|string:name)/keypassphrase/
+
+   Creatre key passphrase for volume `id`.
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      POST /api/v1.0/storage/volume/tank/keypassphrase/ HTTP/1.1
+      Content-Type: application/json
```

Comment:
It'll be great to have example request body as well

Reply:
HTTP body added. I'm not sure about repeating passphrase in this form (it reflects Django form design).
Maybe it's better to pass only one passphrase? What's your opinion @themylogin ?

Reply:
@650elx yes, it's bad API, but as this is temporary solution until we move this to middleware, I think that's fine